types/google-apps-script/apis/classroom_v1.d.ts

```diff
@@ -1,1195 +1,706 @@
-            // Returns a student submission.
-            // Returning a student submission transfers ownership of attached Drive
-            // files to the student and may also update the submission state.
-            // Unlike the Classroom application, returning a student submission does not
-            // set assignedGrade to the draftGrade value.
-            // Only a teacher of the course that contains the requested student submission
-            // may call this method.
-            // This request must be made by the Developer Console project of the
-            // [OAuth client ID](https://support.google.com/cloud/answer/6158849) used to
-            // create the corresponding course work item.
-            // This method returns the following error codes:
-            // * `PERMISSION_DENIED` if the requesting user is not permitted to access the
-            // requested course or course work, return the requested student submission,
-            // or for access errors.
-            // * `INVALID_ARGUMENT` if the request is malformed.
-            // * `NOT_FOUND` if the requested course, course work, or student submission
-            // does not exist.
-            return(resource: any, courseId: string, courseWorkId: string, id: string): void;
-            // Turns in a student submission.
-            // Turning in a student submission transfers ownership of attached Drive
-            // files to the teacher and may also update the submission state.
-            // This may only be called by the student that owns the specified student
-            // submission.
-            // This request must be made by the Developer Console project of the
-            // [OAuth client ID](https://support.google.com/cloud/answer/6158849) used to
-            // create the corresponding course work item.
-            // This method returns the following error codes:
-            // * `PERMISSION_DENIED` if the requesting user is not permitted to access the
-            // requested course or course work, turn in the requested student submission,
-            // or for access errors.
-            // * `INVALID_ARGUMENT` if the request is malformed.
-            // * `NOT_FOUND` if the requested course, course work, or student submission
-            // does not exist.
-            turnIn(resource: any, courseId: string, courseWorkId: string, id: string): void;
-          }
+    namespace Classroom {
+        namespace Collection {
+            namespace Courses {
+                namespace CourseWork {
+                    interface StudentSubmissionsCollection {
+                        // Returns a student submission. * `PERMISSION_DENIED` if the requesting user is not permitted to access the requested course, course work, or student submission or for access errors. * `INVALID_ARGUMENT` if the request is malformed. * `NOT_FOUND` if the requested course, course work, or student submission does not exist.
```

Comment:
The format of the comment here seems worse. It was on multiple lines before, now its all on a single line.

Reply:
Ouch. Didn't notice that. Seems like prettier didn't also clean up comments as all I did was regenerate and remove errors. Will fix.

Reply:
Split the return status comments into multiple lines. Thank you for the input.